tests/models/test_utils.py

```diff
@@ -161,3 +163,21 @@ def test_adding_libraries_to_model_when_version_source_None(sklearn_knn_model):
         assert wheeled_model_info.run_id is not None
         assert wheeled_model_info.run_id != original_run_id
         mlflow_client_mock.assert_called_once_with(model_name, "1")
+
+
+@pytest.mark.parametrize("orient", ["list", "records"])
+def test_schema_enforcement(orient):
```

Comment:
Looks like we already have a lot of enforcement test cases in https://github.com/mlflow/mlflow/blob/master/tests/pyfunc/test_model_export_with_loader_module_and_data_path.py. The suite name definitely doesn't match these cases.

Can we extract these cases into a better named suite and then add this case to it with a more specific name (`test_schema_enforcement` doesn't really tell me what's going on here).

Reply:
refactored and renamed the new suite to what it is: validating pyfunc input types through schema enforcement. The old suite's remaining tests relate to that previous name of the suite.